recipe/meta.yaml

```diff
@@ -19,27 +19,30 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
   host:
     - cython
     - pip
     - python
+    - setuptools >=70.0.0
   run:
     - cython >=0.15.1
-    - Mathics-Scanner >= 1.3.0
+    - Mathics-Scanner >=1.3.0
```

Comment:
This should be >= 1.4.0

Reply:
I am update this now. Finally, we have a release in conda-forge!